entrypoint.sh

```diff
@@ -46,13 +50,18 @@ if [ -n "$INPUT_PRETTIER_PLUGINS" ]; then
             exit 1
         fi
     done
-    npm install --silent --global $INPUT_PRETTIER_PLUGINS
+    npm install --silent $INPUT_PRETTIER_PLUGINS
 fi
+)
 
 echo "Prettifing files..."
 echo "Files:"
 prettier $INPUT_PRETTIER_OPTIONS || echo "Problem running prettier with $INPUT_PRETTIER_OPTIONS"
 
+# Ignore node modules and other action created files
+rm -r node_modules/ || echo "No node_modules/ folder."
+git reset --hard package-lock.json || rm package-lock.json || echo "No package-lock.json file."
+
```

Comment:
I'm surprised you would need to do this...

The `(` + `cd "$GITHUB_ACTION_PATH"`+ `)` above are designed to scope the `npm install` into the action's own directory instead of the client repo.